File: src/main/java/um/tesoreria/report/hexagonal/chequeras/infrastructure/client/dto/ArancelTipoDto.java
package um.tesoreria.report.hexagonal.chequeras.infrastructure.client.dto;

import lombok.Data;

@Data
public class ArancelTipoDto {
    private Integer arancelTipoId;
    private String descripcion = "";
    private Byte medioArancel = 0;
    private Integer arancelTipoIdCompleto;
}

File: src/main/java/um/tesoreria/report/kotlin/model/client/tesoreria/core/ArancelTipoDto.kt
package um.tesoreria.report.kotlin.model.client.tesoreria.core

data class ArancelTipoDto(

    var arancelTipoId: Int? = null,
    var descripcion: String = "",
    var medioArancel: Byte = 0,
    var arancelTipoIdCompleto: Int? = null

)
